sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -22,8 +22,10 @@
 
 const char *stringifyErrorCode(cl_int error);
 
-#define OCL_CODE_TO_STR(code)                                                  \
-  std::string(std::to_string(code) + " (" + stringifyErrorCode(code) + ")")
+static inline  std::string codeToString(cl_uint code){
```

Comment:
```suggestion
static inline std::string codeToString(cl_int code){
```
Double space.

Why cl_uint? stringifyErrorCode expects cl_int.